src/fragments/lib/auth/flutter/signin_with_custom_flow/10_cli_setup.mdx

```diff
@@ -0,0 +1,182 @@
+### How Custom Auth Works
+
+The custom authentication flow supported by Amazon Cognito uses a series of AWS Lambda triggers, which are serverless functions invoked when particular events occur in Cognito. Together, these triggers allow you to establish a series of 'challenges' to which your users must successfully respond in order to authenticate.
+
+The custom authentication flow consists of the following triggers, invoked in order:
+
+- **Define Auth Challenge**: This trigger defines the sequential series of challenges that the user will need to complete in the flow. These challenges can be custom challenges, as well as traditional SRP_A verification.
```

Comment:
Can we maybe elaborate on the SRP_A verification? Even just adding between parenthesis that it is using standard username/password?